### DISCUSSION TITLE
GitHub Copilot could not connect to server.

### DISCUSSION THREAD

**Discussion by b-q** [skills/.github#314] [Code with Codespaces]
For tutorial issue, when codespace opened : 

GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found. You are currently logged

**Comment by sinsukehlab** [comment]
Duplicate of #312
The category should be changed into "Code with Copilot".
Have you enabled GitHub Copilot in the first place?
See: https://docs.github.com/en/copilot
